examples/python/helloworld/greeter_client.py

```diff
@@ -18,17 +18,20 @@
 
 import grpc
 
-import helloworld_pb2
-import helloworld_pb2_grpc
```

Comment:
optional: should we at least keep one example what direct imports generated libraries?

Reply:
If I'm first-time user using gRPC, I would like to learn the best way of using it. So, I agree the hello world example should use runtime imports. I don't have a better name than `helloworld_pregenerated`.